services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy

```diff
@@ -558,7 +577,10 @@ srcLldpDevices=#newSrcEnabled, dstLldpDevices=#newDstEnabled"() {
         }
 
         then: "Device is registered as a flow device"
-        !northbound.getFlowConnectedDevices(flow.id).source.lldp.empty
+        with(northbound.getFlowConnectedDevices(flow.id)) {
+            it.source.lldp.size() == 1
+            verifyEquals(it.source.lldp.first(), lldpData)
+        }
 
         cleanup: "Remove created flow and registered devices, revert switch props"
         northbound.deleteFlow(flow.id)
```

Comment:
do we need `flowHelperV2.deleteFlow(flow.id)` here?

Reply:
yeah we do, but there might be a defect with v2 delete there. Lemme check that tomorrow